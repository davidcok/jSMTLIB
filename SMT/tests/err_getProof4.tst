; get-proof after sat/unknown
(set-option :produce-proofs true)
(set-logic QF_UF)
(assert true)
(check-sat)
(get-proof)
