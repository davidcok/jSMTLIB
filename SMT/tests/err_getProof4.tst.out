unsupported
success
success
sat
(get-proof)
^^^^^^^^^^^
(error "The get-proof command is only valid if :produce-proofs has been enabled")
